Cargo.toml

```diff
@@ -27,7 +27,7 @@ bufstream = "0.1.3"
 imap-proto = "0.16.1"
 nom = { version = "7.1.0", default-features = false }
 base64 = "0.21"
-chrono = { version = "0.4", default-features = false, features = ["std"]}
+chrono = { version = "0.4.24", default-features = false, features = ["std"]}
```

Comment:
Not sure if it is the best way to fix the CI. Look like there is deprecated stuff between `0.4.22` (actual version) and `0.4.24`.